Buildings/Fluid/Movers/UsersGuide.mo

```diff
@@ -437,62 +437,271 @@ If the motor is cooled by the fluid, as indicated by
+as a function of motor part load ratio generated by
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve\">
+Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve</a>.
+If a power curve is provided, the maximum power value on this curve with a 20%
+oversize would be used as default.
+The model then computes the efficiency the same way as the option of
+<code>Values_yMot</code>.
+Also see
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Validation.FlowMachineInterface_yMot\">
+Buildings.Fluid.Movers.BaseClasses.Validation.FlowMachineInterface_yMot</a>
+as an example.
+</li>
+<li>
+<code>NotProvided</code>
+</li>
+</ul>
+<p>
+The table below shows the options available for computing each of the efficiencies.
+For example usage, see <a href=\"Modelica://Buildings.Fluid.Movers.Examples.StaticReset\">
+Buildings.Fluid.Movers.Examples.StaticReset</a>.
 </p>
-
+<table summary=\"summary\" border=\"1\" cellspacing=\"0\" cellpadding=\"2\" style=\"border-collapse:collapse;\">
+<thead>
+  <tr>
+    <th></th>
+    <th><code>PowerCurve</code>*</th>
+    <th><code>EulerNumber</code>*</th>
+    <th><code>Values</code></th>
+    <th><code>Values_yMot</code></th>
+    <th><code>GenericCurve</code></th>
+    <th><code>NotProvided</code></th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td><i>&eta;</i></td>
+    <td>&#10003;</td>
+    <td>&#10003;**</td>
+    <td>&#10003;</td>
+    <td></td>
+    <td></td>
+    <td>&#10003;</td>
+  </tr>
+  <tr>
+    <td><i>&eta;<sub>hyd</sub></i></td>
+    <td>&#10003;</td>
+    <td>&#10003;</td>
+    <td>&#10003;</td>
+    <td></td>
+    <td></td>
+    <td>&#10003;</td>
+  </tr>
+  <tr>
+    <td><i>&eta;<sub>mot</sub></i></td>
+    <td></td>
+    <td></td>
+    <td>&#10003;</td>
+    <td>&#10003;</td>
+    <td>&#10003;</td>
+    <td>&#10003;</td>
+  </tr>
+</tbody>
+</table>
 <p>
-The efficiency data for the motor are a list of points
-<i>V&#775;</i> and <i>&eta;<sub>mot</sub></i>.
+* The <code>PowerCurve</code> and <code>EulerNumber</code> options cannot
+be used for more than one efficiency item.  
+For example, if <i>&eta;<sub>hyd</sub></i> uses <code>EulerNumber</code>,
+then <i>&eta;</i> must use <code>PowerCurve</code>,
+<code>Values</code>, or <code>NotProvided</code>.<br/>
+** Although the Euler number method is defined using <i>&eta;<sub>hyd</sub></i>,
+because the motor efficiency tends to be largely constant from full load down to
+25% - 50% of part motor load (see the documentation of
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve\">
+Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve</a>
+), the relationship between <i>&eta;</i> and
+<i>&eta;<sub>hyd</sub></i> is roughly linear for the range that matters most.
+It is therefore reasonable to approximate that the <code>EulerNumber</code> method
+can also be used to define <i>&eta;</i>.
```

Comment:
Please reformulate (see changes in paper manuscript.)

Reply:
Revised in 6929eb416d5485e6b5cfb2c8001f8c7fbf17b6fb.